app/uk/gov/hmrc/plasticpackagingtax/registration/views/partials/amendment/primaryContactDetails.scala.html

```diff
@@ -18,24 +18,28 @@
 @import uk.gov.hmrc.plasticpackagingtax.registration.models.registration.PrimaryContactDetails
 @import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.subHeading
 @import uk.gov.hmrc.plasticpackagingtax.registration.views.utils.ViewUtils
+@import uk.gov.hmrc.plasticpackagingtax.registration.models.request.JourneyRequest
+@import uk.gov.hmrc.plasticpackagingtax.registration.config.Features.isUkCompanyPrivateBeta
 
 @this(
         subHeading: subHeading,
         govukSummaryList: GovukSummaryList,
         viewUtils: ViewUtils
 )
 
-@(primaryContactDetails: PrimaryContactDetails)(implicit request: Request[_], messages: Messages)
 
+@(primaryContactDetails: PrimaryContactDetails)(implicit request: JourneyRequest[_], messages: Messages)
+
+@isUkCompanyPrivateBetaEnabled = @{!request.isFeatureFlagEnabled(isUkCompanyPrivateBeta)}
 @subHeading(messages("primaryContactDetails.check.label"))
 @govukSummaryList(
     SummaryList(
         rows = Seq(
-            viewUtils.summaryListRow("primaryContactDetails.check.fullName", primaryContactDetails.name, Some(pptRoutes.AmendContactDetailsController.updateContactName())),
-            viewUtils.summaryListRow("primaryContactDetails.check.jobTitle", primaryContactDetails.jobTitle, Some(pptRoutes.AmendContactDetailsController.updateJobTitle())),
-            viewUtils.summaryListRow("primaryContactDetails.check.email", primaryContactDetails.email, Some(pptRoutes.AmendContactDetailsController.updateEmail())),
-            viewUtils.summaryListRow("primaryContactDetails.check.phoneNumber", primaryContactDetails.phoneNumber, Some(pptRoutes.AmendContactDetailsController.updatePhoneNumber())),
-            viewUtils.summaryListRow("primaryContactDetails.check.address", primaryContactDetails.address.map(viewUtils.extractAddress), Some(pptRoutes.AmendContactDetailsController.updateAddress()))
+            viewUtils.summaryListRow("primaryContactDetails.check.fullName", primaryContactDetails.name, Some(pptRoutes.AmendContactDetailsController.updateContactName()), Some(isUkCompanyPrivateBetaEnabled)),
+            viewUtils.summaryListRow("primaryContactDetails.check.jobTitle", primaryContactDetails.jobTitle, Some(pptRoutes.AmendContactDetailsController.updateJobTitle()), Some(isUkCompanyPrivateBetaEnabled)),
+            viewUtils.summaryListRow("primaryContactDetails.check.email", primaryContactDetails.email, Some(pptRoutes.AmendContactDetailsController.updateEmail()), Some(isUkCompanyPrivateBetaEnabled)),
+            viewUtils.summaryListRow("primaryContactDetails.check.phoneNumber", primaryContactDetails.phoneNumber, Some(pptRoutes.AmendContactDetailsController.updatePhoneNumber()), Some(isUkCompanyPrivateBetaEnabled)),
+            viewUtils.summaryListRow("primaryContactDetails.check.address", primaryContactDetails.address.map(viewUtils.extractAddress), Some(pptRoutes.AmendContactDetailsController.updateAddress()), Some(isUkCompanyPrivateBetaEnabled))
```

Comment:
This feels like a strange way to go about doing this. Why not just pass `None` as that already results in the change link being suppressed?

Reply:
No changes should then be needed in `ViewUtils`